openedx/core/djangoapps/programs/tasks.py

```diff
@@ -15,6 +15,7 @@
 from opaque_keys.edx.keys import CourseKey
 
 from common.djangoapps.course_modes.models import CourseMode
+from common.lib.xmodule.xmodule.data import CertificatesDisplayBehaviors
```

Comment:
We just do `from xmodule.data import CertificatesDisplayBehaviors` elsewhere

Reply:
fixed